app/assets/javascripts/application.js

```diff
@@ -11,7 +11,7 @@
 // about supported directives.
 //
 //= require jquery
-//= require bootstrap-sprockets
-//= require jquery_ujs
 //= require jquery-ui
+//= require jquery_ujs
```

Comment:
I know moving these two lines require a lot of research. Thanks for trudging! @njgheorghita